core/src/components/item/item.md.scss

```diff
@@ -154,7 +157,7 @@
   font-size: $item-md-note-slot-font-size;
 }
 
-::slotted(ion-note[slot]) {
+::slotted(ion-note[slot]):not(::slotted([slot="helper"], [slot="error"])) {
```

Comment:
Ah, this is actually a selector list inside of `::slotted()`, which is inside of `:not()`. 

So I think it is fine since slotted has better browser support? https://developer.mozilla.org/en-US/docs/Web/CSS/::slotted

Reply:
Oh right 🤦 This is fine then